Kinvey/Kinvey/Date.swift

```diff
@@ -9,8 +9,8 @@
 import Foundation
 
 extension Date {
-    
-    func toString() -> String {
```

Comment:
Doesn't appear to have been public, but is it possible that current developers can call `toString()` on a `Date` object? Does the `toString()` method have to be deprecated?

Reply:
oh no need since the method was internal. There's no way to call an internal method.

Reply:
I think we can use the `ISO8601DateFormatter` now since we bumped the minimum version on all platforms. Doing that now.

Reply:
it will need to wait until next year since the `milliseconds` is only available in the next minimum bump version, so we need to keep as is for now

https://developer.apple.com/documentation/foundation/iso8601dateformatter/options/2923300-withfractionalseconds